#!/usr/bin/env python
PACKAGE = "cbot_guidance"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lfw_kp",    double_t,    0, "LQR yaw gain Kr", 0.098 ,  0.001, 200.0)
gen.add("lfw_kd",    double_t,    0, "LQR yaw gain Ki", 0.0049 ,  0.001, 200.0)

gen.add("arc_kp",    double_t,    0, "LQR yaw gain Kr", 0.098 ,  0.001, 200.0)
gen.add("arc_kd",    double_t,    0, "LQR yaw gain Ki", 0.0049 ,  0.001, 200.0)

gen.add("guidance_on",    int_t,    0, "Controlller swith", 0 ,  0, 1)

exit(gen.generate(PACKAGE, "Config", "Guidance"))
